queries/tags.scm

```diff
@@ -1,13 +1,19 @@
 (class_declaration
  name: (identifier) @name
- bases: (base_list (_) @name)
  ) @definition.class
 
+(class_declaration
+   bases: (base_list (_) @name)
+ ) @reference.class
```

Comment:
As per https://twitter.com/moke_dev/status/1318551137193250819?s=20, references to parent classes should be references.